<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>171</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="ListWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>333</height>
      </rect>
     </property>
     <property name="accessibleName">
      <string/>
     </property>
     <attribute name="label">
      <string>QListWidget操作</string>
     </attribute>
     <widget class="QToolButton" name="ListWidgetListInt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>初始化列表</string>
      </property>
     </widget>
     <widget class="QToolButton" name="ListWidgetListClr">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>清除列表 </string>
      </property>
     </widget>
     <widget class="QToolButton" name="ListWidgetItemIns">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>插入项</string>
      </property>
     </widget>
     <widget class="QToolButton" name="ListWidgetListAdd">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>添加项</string>
      </property>
     </widget>
     <widget class="QToolButton" name="ListWidgetItemDel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>删除当前项</string>
      </property>
     </widget>
     <widget class="QToolButton" name="ListWidgetItemEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>修改项</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="TreeWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>333</height>
      </rect>
     </property>
     <attribute name="label">
      <string>QTreeWidget操作</string>
     </attribute>
     <widget class="QToolButton" name="TreeWidgetListInt">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>初始化列表</string>
      </property>
     </widget>
     <widget class="QToolButton" name="TreeWidgetItemAdd">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>添加项</string>
      </property>
     </widget>
     <widget class="QToolButton" name="TreeWidgetItemDel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>删除当前项</string>
      </property>
     </widget>
     <widget class="QToolButton" name="TreeWidgetListClr">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>清除列表 </string>
      </property>
     </widget>
     <widget class="QToolButton" name="TreeWidgetItemIns">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>插入项</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="TableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>333</height>
      </rect>
     </property>
     <attribute name="label">
      <string>QTableWidget操作</string>
     </attribute>
     <widget class="QToolButton" name="TableWidgetListInt">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>初始化列表</string>
      </property>
     </widget>
     <widget class="QToolButton" name="TableWidgetItemAdd">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>添加项</string>
      </property>
     </widget>
     <widget class="QToolButton" name="TableWidgetItemDel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>删除当前项</string>
      </property>
     </widget>
     <widget class="QToolButton" name="TableWidgetListClr">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>清除列表 </string>
      </property>
     </widget>
     <widget class="QToolButton" name="TableWidgetItemIns">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>插入项</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="TabWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>591</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="ListWidgetTab">
     <attribute name="title">
      <string>QListWidget</string>
     </attribute>
     <widget class="QListWidget" name="ListWidgetBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>531</width>
        <height>321</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>阅读C++的Qt的epub文件</string>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled|ItemIsTristate</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>阅读Solidity的Epub文件</string>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled|ItemIsTristate</set>
       </property>
      </item>
     </widget>
     <widget class="QCheckBox" name="ListWidgetItemEditable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>是否可编辑</string>
      </property>
     </widget>
     <widget class="QToolButton" name="SelectMenuToolButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>下拉选项</string>
      </property>
     </widget>
     <widget class="QToolButton" name="SelectAllItemsToolButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>全选</string>
      </property>
     </widget>
     <widget class="QToolButton" name="SelectNoItemToolButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>全不选</string>
      </property>
     </widget>
     <widget class="QToolButton" name="SelectReverseCheckStateToolButton">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>10</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>反选</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="TreeWidgetTab">
     <attribute name="title">
      <string>QTreeWidget</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="TableWidgetTab">
     <attribute name="title">
      <string>QTableWidget</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="MainToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="SecondaryToolBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <action name="ActionListInt">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>初始化列表</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="ActionListClr">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>清除列表</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="ActionItemIns">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>插入项</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="ActionItemAdd">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>添加项</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="ActionItemDel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>删除当前项</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="ActionSelectAll">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>全选</string>
   </property>
   <property name="toolTip">
    <string>全选</string>
   </property>
   <property name="shortcut">
    <string>Shift+A</string>
   </property>
  </action>
  <action name="ActionAllNotSelected">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>全不选</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="ActionReverseSelected">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>反选</string>
   </property>
   <property name="toolTip">
    <string>反选</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="ActionQuit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="ActionItemEdit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>修改项</string>
   </property>
   <property name="toolTip">
    <string>修改项</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
